README.md

```diff
@@ -8,28 +8,33 @@
 
 require in the SDK:
 
-`const OneblinkSDK = require('@oneblink/sdk')`
+````javascript
+const OneblinkSDK = require('@oneblink/sdk')
+```
 
 initialise the SDK object:
 
 ```javascript
-const oneblink = new OneblinkSDK()
+const FORMS_KEY = '12345'
+const FORMS_SECRET = 'abcd'
+const oneblink = new OneblinkSDK({
+  formsAccessKey: FORMS_KEY,
+  formsSecret: FORMS_SECRET
```

Comment:
if this is suppose to be a generic sdk, we prob want this variables to be called `accessKey` and `secretKey`, no `forms`.